src/kubernetes-ingress-controller/guides/using-kongingress-resource.md

```diff
@@ -5,6 +5,7 @@ title: Using KongIngress resource
 In this guide, we will learn how to use KongIngress resource to control
 proxy behavior.
 
+{% if_version lt:2.8.x -%}
```

Comment:
```suggestion
{% if_version lte:2.7.x -%}

```
I ran into this problem the other day, I think the reason why it might not be displaying correctly is because there isn't a blank line before the note.

Reply:
Oh, and `lte` and `gte` both _include_ the listed version (in addition to the less than and greater than versions), so both of these notes would show for 2.8.x based on how they are currently configured.

Maybe this one should be `2.7.x`?